ui/App/UserProfileScreen.svelte

```diff
@@ -30,14 +43,105 @@
       },
     });
   }
+
+  let registration: Registration | undefined;
+  let ownedOrgs: Org[] = [];
+
+  const wallet = svelteStore.get(Wallet.store);
+  const ethereumEnvironment = ethereum.selectedEnvironment;
+
+  async function loadSidebarData(): Promise<void> {
+    const address = user.metadata.ethereum?.address;
+    console.log(user);
+    if (!address) {
+      return;
+    }
+
+    try {
+      const gnosisSafeWallets = await Safe.getSafesByOwner(
+        wallet.environment,
+        address
+      );
+      ownedOrgs = await graph.getOwnedOrgs([address, ...gnosisSafeWallets]);
+      if (ownedOrgs) {
+        ownedOrgs.map(async org => {
+          const registration = await getCachedRegistrationByAddress(org.id);
+          if (registration) {
+            org.registration = registration;
+          }
+          return org;
+        });
+      }
+    } catch (err: unknown) {
+      notification.showException(
+        new error.Error({
+          code: error.Code.ProjectRequestFailure,
+          message: "Failed to fetch orgs for sidebar.",
+          source: err,
+        })
+      );
+    }
+
+    try {
```

Comment:
Same with this `try/catch`.

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/2559/commits/609defab41cd8bb1efa0932c158dc954f9a87ab4